src/sentry/demo/data_population.py

```diff
@@ -732,25 +821,31 @@ def generate_releases(self, projects):
 
             release_time += timedelta(hours=hourly_release_cadence)
 
-    def generate_alerts(self, project):
-        self.generate_metric_alert(project)
+    def generate_alerts(self, project, warning, critical):
+        self.generate_metric_alert(project, warning, critical)
         self.generate_issue_alert(project)
 
-    def generate_metric_alert(self, project):
+    def generate_metric_alert(self, project, warning_threshold, critical_threshold):
         org = project.organization
         team = Team.objects.filter(organization=org).first()
+        time_interval = 60
```

Comment:
@roggenkemper might be good to make this a constant we can modify through the config